content/non-web-ict.md

```diff
@@ -55,47 +57,39 @@ Quick link: [WCAG2ICT Note](https://www.w3.org/TR/wcag2ict/)
 
 *This document:* **WCAG2ICT** is a W3C Working Group Note that is informative — it is not normative and does not set requirements. It provides guidance on the application of WCAG 2.0 principles, guidelines, and success criteria to non-web documents and software. WCAG2ICT was developed to provide an authoritative interpretation of how WCAG 2.0 can apply in different contexts. For information on the scope of WCAG2ICT, see the [WCAG2ICT Introduction](https://www.w3.org/TR/wcag2ict/#intro).
 
-*Note:* **WCAG2ICT is still relevant for later versions of WCAG 2**. For example, WCAG 2.1 does not change the requirements of WCAG 2.0 that are addressed in WCAG2ICT; WCAG 2.1 adds additional requirements. W3C may update WCAG2ICT in the future to add the additional requirements from later versions of WCAG 2.
+*Note:* **WCAG2ICT is still relevant for later versions of WCAG 2**. For example, WCAG 2.1 does not change the requirements of WCAG 2.0 that are addressed in WCAG2ICT; WCAG 2.1 adds additional requirements. In 2022, the WCAG2ICT Task Force is updating WCAG2ICT to cover WCAG 2.1 and WCAG 2.2.
```

Comment:
Right. I thought good to put "2022" to let readers know this is current work. I didn't intend for it to convey that it would be finished in 2022. We can update this as work progresses. For example, if we add link to Editors' draft with 2022 date, then we can delete "In 2022, " here.

Reply:
I deleted it. changed to "currently" -- and the footer has the date this page was updated, so hopefully that covers it :)